mautic/mautic
# Fix group by If you use count columns for assets download 2

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This is regression issue after merge https://github.com/mautic/mautic/pull/10693



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create assets: **test 1** and **test 2**
3. Open download link for `test 1` in incognito window
4. Close incognito window and do the same for asset `test 1` again
5. Then close incognito window and open download link for `test 2` in incognito window
6.   Now you should have 2 downloads for `test 1` and 1 download for `test 2`
7. Create report for assets and choose alias and contact ID as columns
8. You should see 3 rows in report
9. Then add download count and unique download count
10. You should see just two lines in report - what is wrong, because this is report of assets downloads.
11. After PR you should see 3 rows in report
12. If you want report by assets, then you can add group by by Asset ID and you should see two lines, what is correct




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10833"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
